### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by PriyaVarra**
The final step of setting up the Github Repo is to have a host review my change to README.md and merge my pull request with the master branch.

Thanks!

### CODE REVIEW

**Comment by PriyaVarra** [comment]
Rearranged structure of portfolio and added information, images, and links for project section of portfolio.

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,185 @@
...
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+<html lang="en">
```
Comment (wjiamin) [inline portfolio/src/main/webapp/index.html:2]: These formatting looks a bit odd, is there a way to fix it with indentation?

Comment (PriyaVarra) [reply portfolio/src/main/webapp/index.html:2]: I will make sure to check all of the indentation in my current code. My current version does not contain this piece of code anymore.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,185 @@
...
     </div>
-  </body>
+
+    <div class="w3-col m4 w3-margin-bottom">
+      <div class="w3-light-grey">
+        <img src="images/Figure2.png" alt="wbc" style="width:100%">
+        <div class="w3-container">
+          <h4> White Blood Cell Image Classification </h4>
+          <p class="w3-opacity">Computational Biology - Spring 2019 &nbsp <a href="https://github.com/PriyaVarra/Classifying-White-Blood-Cell-Images-Using-Deep-Learning"> <img src="images/GitHub-Mark-32px.png" width="32" height="32"> </a> </p>
+          <p> In this project, I attempted to create a computational model capable of classifying an image of a stained white blood cell into one of the four major types of white blood cells. Specifically, I used Python to preprocess a set of publicly available training images before using Tensorflow and Keras to create, train, and test three convolutional neural networks with the best model achieving an accuracy of 78%.   </p>
```
Comment (wjiamin) [inline portfolio/src/main/webapp/index.html:107]: I am not sure if this is just not being translated properly in my review page, but would it be possible to break this long line into multiple lines for easier reading?

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -1,18 +1,185 @@
...
+  <!-- Gallery -->
+  <div class="w3-container" style="margin-top:80px" id="gallery">
+    <h1 class="w3-xxxlarge w3-text-teal"><b>Gallery</b></h1>
+    <hr style="width:50px;border:5px solid teal" class="w3-round">
+  </div>
+  
+
+  <div class="w3-row-padding">
+    <div class="w3-half">
+      <img src="images/myAvatar.png" style="width:100%" onclick="onClick(this)" alt="Concrete meets bricks">
```
Comment (wjiamin) [inline portfolio/src/main/webapp/index.html:156]: Not necessary but it is always nice to break up long lines: https://google.github.io/styleguide/htmlcssguide.html#HTML_Line-Wrapping

File: portfolio/src/main/webapp/index.html [hunk 4]
```diff
@@ -1,18 +1,185 @@
...
+
+
+  <!-- About Me -->
+  <div class="w3-container" id="about" style="margin-top:75px">
+  	<h1 class="w3-jumbo"><b>Welcome to My Portfolio!</b></h1>
+    <h1 class="w3-xxxlarge w3-text-teal"><b>About Me</b></h1>
+    <hr style="width:50px;border:5px solid teal" class="w3-round">
+    <div class="w3-row">
+         <div class="w3-col m3 w3-margin-bottom">
+            <img style='border: 15px solid #008080' src='images/web_pic.jpg'  height = "350">
```
[the remark this answers is not in the document]
Comment (PriyaVarra) [reply portfolio/src/main/webapp/index.html:58]: Thanks for the suggestion! This has been changed in the most recent version.

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (PriyaVarra) [reply portfolio/src/main/webapp/index.html:107]: I completely agree. I have fixed this in the most recent version.

File: portfolio/src/main/webapp/index.html (hunk 3 above)
Comment (PriyaVarra) [reply portfolio/src/main/webapp/index.html:156]: Thanks for the suggestion! I will check this out in my most recent version.

**Pull request merged**